src/Kdyby/Events/Diagnostics/Panel.php

```diff
@@ -388,7 +388,7 @@ private function renderListeners($ids)
 
 		$h = 'htmlspecialchars';
 
-		$shortFilename = function (\ReflectionMethod $refl) {
+		$shortFilename = function (\ReflectionFunctionAbstract $refl) {
```

Comment:
Callback::toReflection returns only ReflectionMethod or ReflectionFunction. And Reflector do not have getFilename, getStartLine etc.